sample-app/build.gradle.kts

```diff
@@ -33,6 +33,11 @@ android {
     buildFeatures {
         viewBinding = true
     }
+    packagingOptions {
+        jniLibs {
+            useLegacyPackaging = true
+        }
+    }
```

Comment:
This is to support the 16KB page size in NDK27.
- https://developer.android.com/guide/practices/page-sizes#compile-16-kb-alignment